### DISCUSSION TITLE
Why doesn't the simple airfoil example wing tip position plot start at zero?

### DISCUSSION THREAD

**Discussion by wmschy** [ImperialCollegeLondon/sharpy#145] [Q&A] [answered]
When I ran the simple_airfoil Jupyter notebook, in the "DynamicCoupled" solver of this example, I noticed that the starting beam position data (from the first timestep) is not the same as what is initialized in the Beam object, but rather, the first timestep originates in the "StaticCoupled" solver, which precedes it in the flow.
Also, my wing tip plot does not start at zero, as I expected (same for the plot image from the example's online documentation).
When I remove "staticcoupled" from the flow, the wing tip plot starts at zero and mirrors the other plot (below), (and the initial Beam position data is the same as what is initialized the Beam generate() method), as I expected.

Is this behavior intentional; that is, should StaticCoupled affect the results of DynamicCoupled when placed before it in the flow? If so, what is the rationale (i.e., what effect or benefit is there in combining them (other than it's easier than running them separately))?

My wing tip plot (excluding "StaticCoupled"), which starts at zero, as opposed to the plot from the example's documentation:
![Screenshot from 2021-04-22 12-46-23](https://user-images.githubusercontent.com/40265420/115708497-5ef96780-a370-11eb-8006-eaf8a7137bb8.png)

Thank you

**Comment by ArturoMS13** [comment, marked as answer]
Dear wmschy,

The solvers included in the `flow` variable run sequentially and use the solution from the previous solver. For example, `StaticCoupled` uses the model that `BeamLoader` and `AerogridLoader` have created.

The simple airfoil case is a cantiliver beam subjected to aerodynamic forces. The idea behind it is finding the equilibrium position for that cantilever beam (with `StaticCoupled`) and then change the angle of attack (with respect to the equilibrium position) with `DynamicCoupled`. 

If you think of this case as an airplane wing: `StaticCoupled` would give you the cruise deformation (which is not zero as you noted) and `DynamicCoupled` would simulate a sudden change in the angle of attack.

Running `DynamicCoupled` without `StaticCoupled` is possible if that fits the real case you want to simulate. However, be aware that sudden changes might lead to numerical convergence problems.

I hope this helps.
